docs/application/dotnet/samples/ui/overview.md

```diff
@@ -193,62 +193,6 @@ for TD:
 			<p>This sample application demonstrates how to use various kinds of Tizen Wearable UI controls using the Xamarin.Forms and <a href="https://samsung.github.io/Tizen.CircularUI/api/index.html" target="_blank">Tizen.Wearable.CircularUI</a>.</p>
 			</td>
 		</tr>
-		<tr>
-			<td><img alt="" height="180" src="media/w46_animationsample_nui.png" width="180"/></td>
-			<td>
-			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/NUI/AnimationsSample" target="_blank"><strong>(W) AnimationSample (NUI)</strong></a></p>
```

Comment:
Was it verified if these samples are referenced in any other files, I did find some of them referred in other guides however they seem to orphan guides not appearing in the TOC.

Reply:
I heard some news from NUI team one month ago. They have a plan revising NUI document. It might be already done or under construction... hm.. anyway, I will ask them to remove invalid sample link from NUI guide.